aten/src/ATen/native/native_functions.yaml

```diff
@@ -296,7 +296,7 @@
 
 - func: _shape_as_tensor(Tensor self) -> Tensor
 
-- func: dropout(Tensor input, float p, bool train) -> Tensor
+- func: dropout(Tensor(a) input, float p, bool train) -> Tensor(a)
```

Comment:
does this mean the output tensor is optionally aliased with `input`?

Reply:
I think so

Reply:
I feel like this won't work for QAT because there will be no way to pattern match anymore....

Reply:
yeah wondering if we can decompose this to a different op/ops for QAT, although the decomposition might change the semantics